src/Defra.PTS.Web.UI/Views/Shared/_Layout.cshtml

```diff
@@ -1,6 +1,14 @@
-﻿@using Defra.PTS.Web.UI.Constants;
+﻿@using Defra.PTS.Web.UI.Constants
+@inject Microsoft.AspNetCore.Http.IHttpContextAccessor context
+
+@{
+    var locale = context.HttpContext.Request.Cookies["locale"] ?? "en";
+    var toggleButtonText = (locale == "en") ? "Cymraeg" : "English";
```

Comment:
again not sure why this is here

Reply:
It Retrives the locale from the cookies defaulting to 'en'.
Used ViewData to use locale elsewhere in the View if required
.